PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -80,7 +82,7 @@ def _generate_loader_cache(gdk_pixbuf_query_loaders, libdir, loader_libs):
     #
     # On Windows, the loaders lib directory is relative, starts with 'lib', and uses \\ as path separators
     # (escaped \).
-    cachedata = compat.exec_command_stdout(gdk_pixbuf_query_loaders, *loader_libs)
+    cachedata = run([gdk_pixbuf_query_loaders, *loader_libs], check=True, stdout=PIPE, encoding='utf-8').stdout
```

Comment:
I would prefer if these were `subprocess.run` and `subprocess.PIPE` to immediately make it clear where they are coming from. Applies to both changes from `compat.exec_command_stdout`.

Reply:
Fixed!